loadtest/user/userentity/report.go

```diff
@@ -0,0 +1,68 @@
+package userentity
+
+import (
+	"context"
+	"math/rand"
+	"time"
+
+	"github.com/mattermost/mattermost/server/public/model"
+)
+
+var (
+	userAgents = []string{"desktop", "firefox", "chrome", "safari", "edge", "other"}
+	platforms  = []string{"linux", "macos", "ios", "android", "windows", "other"}
```

Comment:
Let's just use other/other for this. If we use all the platforms, then this becomes hard for a user to open a browser session and observe the "actual" client metrics in a live load test.

More context here: https://community.mattermost.com/core/pl/9y8cgyxxq7yq5yjorpqykbjkrw

Reply:
Thank you for bringing this up. I think this approach will strike good balance between team wanting to use load testing tool for the web client metrics and tool being used to measure impact of web client metrics with load test.